src/MatrixAlgebraKit.jl

```diff
@@ -3,16 +3,16 @@ module MatrixAlgebraKit
 using LinearAlgebra: LinearAlgebra
 using LinearAlgebra: norm # TODO: eleminate if we use VectorInterface.jl?
 using LinearAlgebra: mul!, rmul!, lmul!, adjoint!, rdiv!, ldiv!
-using LinearAlgebra: sylvester
-using LinearAlgebra: isposdef, issymmetric
+using LinearAlgebra: sylvester, lu!
+using LinearAlgebra: isposdef, issymmetric, opnorm
```

Comment:
```suggestion
using LinearAlgebra: isposdef, issymmetric
```
I don't think this is being used (anymore)?

Reply:
No, that's another remnant of my github rebase struggles.